spek-runtimes/common/src/main/kotlin/org/spekframework/spek2/runtime/FixturesAdapter.kt

```diff
@@ -4,19 +4,17 @@ import org.spekframework.spek2.lifecycle.ActionScope
 import org.spekframework.spek2.lifecycle.GroupScope
 import org.spekframework.spek2.lifecycle.LifecycleListener
 import org.spekframework.spek2.lifecycle.TestScope
-import java.util.LinkedHashMap
-import java.util.LinkedList
 
 /**
  *
  * @author Ranie Jade Ramiso
  */
 class FixturesAdapter: LifecycleListener {
-    private val beforeEachTest: LinkedHashMap<GroupScope, LinkedList<() -> Unit>> = LinkedHashMap()
-    private val afterEachTest: LinkedHashMap<GroupScope, LinkedList<() -> Unit>> = LinkedHashMap()
+    private val beforeEachTest: LinkedHashMap<GroupScope, MutableList<() -> Unit>> = LinkedHashMap()
+    private val afterEachTest: LinkedHashMap<GroupScope, MutableList<() -> Unit>> = LinkedHashMap()
```

Comment:
huh, `LinkedHashMap` is implemented for js and native runtimes but `LinkedList` not?

Reply:
It's not implemented https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/index.html.

Reply:
That's funny :)